Subject: default-config-no-gssapi.patch for OpenPrinting cups 2.4.20
Description: Alpine aports local configuration; disable GSSAPI in default cupsd.conf.
Upstream source: https://github.com/alpinelinux/aports/blob/master/main/cups/default-config-no-gssapi.patch
Base tree: OpenPrinting cups v2.4.20
Files: conf/cupsd.conf.in
Verbatim copy from alpinelinux/aports master main/cups.

Upstream: Not applicable (local configuration)
Reason: Sets config defaults for Alpine

--- a/conf/cupsd.conf.in
+++ b/conf/cupsd.conf.in
@@ -176,12 +176,12 @@ IdleExitTimeout @EXIT_TIMEOUT@
 
   # Job-related operations must be done by the owner or an administrator...
   <Limit Create-Job Print-Job Print-URI Validate-Job>
-    AuthType Negotiate
+    AuthType Default
     Order deny,allow
   </Limit>
 
   <Limit Send-Document Send-URI Hold-Job Release-Job Restart-Job Purge-Jobs Set-Job-Attributes Create-Job-Subscriptions Renew-Subscription Cancel-Subscription Get-Notifications Reprocess-Job Cancel-Current-Job Suspend-Current-Job Resume-Job Cancel-My-Jobs Close-Job CUPS-Move-Job CUPS-Get-Document>
-    AuthType Negotiate
+    AuthType Default
     Require user @OWNER @SYSTEM
     Order deny,allow
   </Limit>
@@ -202,7 +202,7 @@ IdleExitTimeout @EXIT_TIMEOUT@
 
   # Only the owner or an administrator can cancel or authenticate a job...
   <Limit Cancel-Job CUPS-Authenticate-Job>
-    AuthType Negotiate
+    AuthType Default
     Require user @OWNER @CUPS_DEFAULT_PRINTOPERATOR_AUTH@
     Order deny,allow
   </Limit>
@@ -214,7 +214,7 @@ IdleExitTimeout @EXIT_TIMEOUT@
 
   # Require authorization for any operation not listed
   <Limit All>
-    AuthType Negotiate
+    AuthType Default
     Order deny,allow
   </Limit>
 </Policy>
